trojsten/special/plugin_prask_7_1_1/contants.py

```diff
@@ -0,0 +1,41 @@
+class Level:
+    def __init__(self, url, source_fname, hint):
+        self.url = url
+        self.source_fname = source_fname
+        self.hint = hint
+
+
+LEVELS = [
+    Level("letsplay", "pexeso.html", "Je to skoro ako pexeso."),
+    Level("level2", "urlchange.html", "Pozri sa na URL aktuálnej stránky :)"),
+    Level("level3", "insource.html", "Ak heslo nevidno na stránke, mohlo sa zašiť niekam do kódu."),
+    Level("malina", "morse.html", "Jeden pes je dlhý, druhý zase gulatý. Nie sú to Morseho psi?"),
+    Level(
+        "woof",
+        "button.html",
+        (
+            "Prefix ukazoval, že cez Developer Tools (F12) môžeme meniť stránku. Skús tak spraviť "
```

Comment:
nechyba tu ciarka? nie je to nahodou RTE?

Reply:
Hmm, imho sú tam čiarky v pohode. Čo si myslel tým RTE?